edgelet/iotedged/src/lib.rs

```diff
@@ -1357,7 +1357,7 @@ where
     M::Settings: Serialize,
     C: CreateCertificate + GetIssuerAlias + MasterEncryptionKey,
 {
-    // Remove all edge containers and destroy the cache (settings and dps backup)
+    // Remove all edge containers and delete settings state
```

Comment:
I would remove "delete settings state" from this as well, since there's no explicit call to delete the file.

Reply:
True, will do.

Reply:
Done